bench-exchange/src/cli.rs

```diff
@@ -44,10 +44,10 @@ impl Default for Config {
     }
 }
 
-pub fn build_args<'a, 'b>() -> App<'a, 'b> {
+pub fn build_args<'a, 'b>(version: &'b str) -> App<'a, 'b> {
```

Comment:
As this looks odd to be added, however, this is purely artificial argument needed to make rustc happy about lifetimes of `solana_clap_utils::version!()`. The problem is that the returned type of the procedure macro is `&str` and is treated as a `temporary value` if directly called inside this function, resulting in this following error:

```
error[E0515]: cannot return value referencing temporary value
   --> bench-exchange/src/cli.rs:48:5
    |
48  | /     App::new(crate_name!())
49  | |         .about(crate_description!())
50  | |         .version(solana_clap_utils::version!())
    | |                  ----------------------------- temporary value created here
51  | |         .arg(
...   |
162 | |                 .help("Read client keys and stakes from the YAML file"),
163 | |         )
    | |_________^ returns a value referencing data owned by the current function
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

```
I've researched this issue abit, but didn't come up with a better solution.